src/api/idenfy/types.ts

```diff
@@ -0,0 +1,65 @@
+export interface IdenfyTokenRequest {
+  readonly clientId: string;
+  readonly additionalData?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+  readonly additionalSteps?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+  readonly address?: string | null;
+  readonly callbackUrl?: string;
+  readonly country?: string | null;
+  readonly dateOfBirth?: string | null;
+  readonly dateOfExpiry?: string | null;
+  readonly dateOfIssue?: string | null;
+  readonly documentNumber?: string | null;
+  readonly documents?: string[];
+  readonly errorUrl?: string;
+  readonly expiryTime?: number;
+  readonly externalRef?: string | null;
+  readonly firstName?: string;
+  readonly generateDigitString?: boolean;
+  readonly lastName?: string;
```

Comment:
along with the required `clientId` that we send to them, we can also send the optional `firstName` `lastName` fields. This would ensure the person doing the verification session has the same name as the profile. Maybe we can do that post MVP?